docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -0,0 +1,102 @@
+---
+id: espresso
+title: "Configuration Syntax: Espresso"
+sidebar_label: Espresso
+---
+
+import Highlight from '../../../src/components/highlight.jsx'
+
+Please refer to the [Common Configuration Syntax Reference](/testrunner-toolkit/configuration/common-syntax)for information regarding fields such as `apiVersion`, `kind`, `suites`, `sauce`, etc.
+
+## Example Configuration
+
+```yaml reference
+https://github.com/saucelabs/saucectl-espresso-example/blob/master/.sauce/config.yml
+```
+
+## `espresso`
+
+__Description__: Details specific to the `espresso` configuration.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+espresso:
+  app: ./apps/calc.apk
+  testApp: ./apps/calc-success.apk
+```
+
+### `app`
+
+__Description__: Path to the application.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  app: ./apps/calc.apk
+```
+
+### `testApp`
+
+__Description__: Path to the testing application.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  testApp: ./apps/calc-success.apk
+```
+
+## `devices`
+
+__Description__: Field for defining devices details such as the device name, orientation, and platformVersions.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+devices:
+  - name: "Android GoogleApi Emulator"
+    orientation: portrait
+    platformVersions:
+      - "11.0"
+      - "10.0"
+```
+
+### `name`
+
+__Description__: Name of the device. All supported devices can be found by following this [link](https://app.saucelabs.com/live/web-testing/virtual) 
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  - name: "Android GoogleApi Emulator"
+```
+
+### `orientation`
+
+__Description__: Screen orientation. 
+
+__Type__: *string*
```

Comment:
Is this a string or an enum?

Reply:
orientation is either `portrait` or `landscape`, it can be treated as a enum

Reply:
I changed this field to the `enum` and also added possible values.